src/napari_spatialdata/_widgets.py

```diff
@@ -379,6 +493,34 @@ def cmap(self) -> str:
         return self._cmap
 
 
+class MatplotlibWidget(NapariMPLWidget):
+    def __init__(self, viewer: Viewer, model: ImageModel):
+
+        super().__init__(viewer)
+
+        self.viewer = viewer
+        self.model = model
```

Comment:
how is self.model used in the other widgets, is it `_model` or `model` ? Please align all of them, and same for viewer.

Reply:
It varies among the widgets. For example, in class ListWidget self._viewer is used and in class AListWidget self.viewer is used. Should I change all of them to the first example i.e `self._viewer = viewer`.

Reply:
yes please, feel free to do it everywhere if appropriate. Maybe add it as a property